extensions/extension-rest-api-call/src/main/resources/DeviceAttributeRestDescriptor.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Do we need this action as part of this story Or have you mistakenly committed this ?

Reply:
By mistake I have done that. It is not in the latest commit.